### DISCUSSION TITLE
Alpha 154

### DISCUSSION THREAD

**Discussion by Eugeny** [Eugeny/tabby#4434] [Release Notes]
This release consists mostly of minor bug fixes

- The legacy `hterm` frontend has been removed
- Profile colours now show up in the profile selector
- Fixed instantiating saved layouts 
- Custom tab titles are now recovered after a restart
- Fixed jump hosts not inheriting default profile settings
- Fixed titlebar not adjusting to macOS fullscreen mode
- macOS: don't exit when the last window is closed
- Profiles: color field now autocompletes
- Telnet: fixed window size negotiation message order

<img src="https://ga-beacon.appspot.com/UA-3278102-18/github/release-153" width="1"/>

<hr /><em>This discussion was created from the release <a href='https://github.com/Eugeny/tabby/releases/tag/v1.0.154'>Alpha 154</a>.</em>

**Comment by parampreetss** [comment]
Hi. I have a couple of observations in the Linux app (.deb). Don't know whether these can be termed as bugs, as I don't know how to reproduce them. They appear randomly. So seeking inputs here:

1. After I close Tabby, sometimes its processes continue running in the background. And I can't launch another Tabby window, until I kill those processes.
2. On a number of occasions, when I paste text into terminal with hotkeys, it pastes the clipboard contents twice with a single keypress. So each pasted command or path appears twice in the terminal, without any line breaks in between.

Also, another thing that I find different in case of window appearance, the color of the active tab doesn't match the terminal background. Rather, the inactive tabs' color matches the terminal background. This is totally opposite of what we are normally used to (like in Chromium, Sublime Text, Atom editor etc), where inactive tabs appear in a different color. I understand aesthetics can be subjective. Just sharing my opinion.

**Reply by Eugeny** [reply]
Thanks for the feedback - the #1 has just been fixed in v155 a few minutes ago, and #2 depends on the theme you're using and the "terminal background" setting in Settings -> Appearance. With stock themes, and "terminal background = from theme" (default), terminal background will match the active tab's color - like on this README screenshot: ![](https://github.com/Eugeny/tabby/raw/master/docs/readme.png)

**Comment by parampreetss** [comment]
Thanks Eugene. I installed v155, and #1 is indeed gone.
But this multiple paste problem is still there. Would you be able to advise me on that, or should I report a bug? In the attached screenshot, I pasted the command from a web page into the Tabby terminal. I tried twice and you'll see in both lines, the command that I pasted appears twice. I pasted in Konsole, Kate and Sublime text, there only a single copy is pasted.![Screenshot_20210819_162712](https://user-images.githubusercontent.com/20267899/130057959-711f4ad6-fda3-4fde-8559-3590e31a3cfa.png)

**Reply by Eugeny** [reply]
Yes - please paste is separately

**Reply by parampreetss** [reply]
Sorry, I didn't understand that. Did you mean I should report a bug?

**Reply by Eugeny** [reply]
Sorry, edited my reply into a mess :D I meant post it separately, thanks!
